File: src/test/java/com/ject/studytrip/stamp/fixture/UpdateStampOrderRequestFixture.java
package com.ject.studytrip.stamp.fixture;

import com.ject.studytrip.stamp.presentation.dto.request.UpdateStampOrderRequest;
import java.util.List;

public class UpdateStampOrderRequestFixture {
    private List<Long> orderedStampIds = List.of(1L, 2L);

    public UpdateStampOrderRequest buildUpdateOrders() {
        return new UpdateStampOrderRequest(orderedStampIds);
    }

    public UpdateStampOrderRequestFixture withOrderedStampIds(List<Long> orderedStampIds) {
        this.orderedStampIds = orderedStampIds;
        return this;
    }
}

File: src/test/kotlin/com/ject/studytrip/stamp/fixture/UpdateStampOrderRequestFixture.kt
package com.ject.studytrip.stamp.fixture

import com.ject.studytrip.stamp.presentation.dto.request.UpdateStampOrderRequest

class UpdateStampOrderRequestFixture(
    private val orderedStampIds: List<Long> = listOf(1L, 2L),
) {
    fun withOrderedStampIds(orderedStampIds: List<Long>): UpdateStampOrderRequestFixture = UpdateStampOrderRequestFixture(orderedStampIds)

    fun build(): UpdateStampOrderRequest = UpdateStampOrderRequest(orderedStampIds)
}
